source/utilities.tex

```diff
@@ -18341,10 +18341,25 @@
 if there is a template type argument $\tcode{B}_{i}$
 for which \tcode{bool($\tcode{B}_{i}$::value)} is \tcode{false},
 then instantiating \tcode{conjunction<$\tcode{B}_{1}$, $\dotsc$, $\tcode{B}_{N}$>::value}
-does not require the instantiation of \tcode{$\tcode{B}_{j}$::value} for $j > i$.
+does not require a definition of \tcode{$\tcode{B}_{j}$::value} to exist\iref{temp.inst} for $i < j \leq N$.
 \begin{note} This is analogous to the short-circuiting behavior of
 the built-in operator \tcode{\&\&}.
 \end{note}
+\begin{example}
+This allows types without a \tcode{value} static data member
+to be template arguments for \tcode{conjunction}, as long as an
+earlier argument evaluates to \tcode{false}, e.g.
```

Comment:
And this with ", for example:". Otherwise you have a dangling sentence.

Reply:
Or maybe even  turn the example into a note and turn the example-in-the-example into an example? And then merge the note with the previous note?

Reply:
I'd prefer to keep it separate from the previous Note. The run-time short-circuiting of `&&` doesn't allow you to have invalid expressions on the RHS or functions that aren't defined, which is the property being demonstrated by the example. It's analogous, but not actually the same.

Reply:
Change "e.g." to "for example:" and changed the Example to a second Note.